airqo-platform/AirQo-api
# Add Design to welcome Email messages

**_WHAT DOES THIS PR DO?_**
- Add Design to welcome Email messages

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [MOB-199](https://airqoteam.atlassian.net/browse/MOB-199)
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
- Create an account on the mobile app using an email.
- You should get an email after 5 minutes

**_Screenshots
![2](https://user-images.githubusercontent.com/86492979/234783806-e6713eda-321b-4053-877e-3d3fee4f1cce.jpg)





[MOB-199]: https://airqoteam.atlassian.net/browse/MOB-199?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
